packages/gasket-utils/lib/package-manager.js

```diff
@@ -114,6 +130,19 @@ class PackageManager {
         cwd: this.dest,
         env
       });
+    } else if (this.manager === 'pnpm') {
+      const argv = [cmd].concat(args);
+
+      //
+      // Support for the .npmrc configured via --npmconfig flag.
+      // pnpm does not have a "userconfig" CLI flag, it does however still
+      // support the npm_config_* environment variables for npm compatibility.
+      if (this.npmconfig) env.NPM_CONFIG_USERCONFIG = this.npmconfig;
```

Comment:
The `--npmconfig` flag is being removed in the next major version. See https://github.com/godaddy/gasket/pull/647

While this is not a breaking change, if there's no immediate need for it now, could we just save `pnpm` support for our next major version 7?

Reply:
I don't see a need for it. people can migrate from npm to pnpm pretty easily. I will update the PR to merge to v7 base